scripts/playMode/handleTurnChangeEffects.js

```diff
@@ -1,17 +1,31 @@
 function handleTurnChangeEffects({ turnToggle = false } = {}) {
 	// move this mapping outside
 	const popoverTextMapping = {
-		one: [playerNameLabelMapping.one],
-		two: [playerNameLabelMapping.two, 'orange'],
+		one: [PLAYER_NAME_LABEL_MAPPING.one],
```

Comment:
update this maaping and move this mapping to constants

Reply:
done.
object mapping converted to function with switch expression
removed playModes.css
inheriting ball box-shadow from ball current color